src/software/thunderscope/log/g3log_widget.py

```diff
@@ -37,12 +41,20 @@ def refresh(self):
         if not log:
             return
 
-        log_str = "{} {} [{}->{}] {}\n".format(
-            log.created_timestamp.epoch_timestamp_seconds,
-            log.log_level,
-            log.file_name,
-            log.line_number,
-            log.log_msg,
-        )
-
-        self.write(log_str)
+        # Checks whether this type of log is enabled from checkboxes
+        if (
+            (log.log_level == 0 and self.checkboxWidget.isChecked1())
+            or (log.log_level == 1 and self.checkboxWidget.isChecked2())
+            or (log.log_level == 2 and self.checkboxWidget.isChecked3())
+            or (log.log_level == 3 and self.checkboxWidget.isChecked4())
```

Comment:
we should use LogLevel.INFO and LogLevel.DEBUG and whatnot instead of 0 or 1

the protobuf enum should be generated into python

try importing `from proto.robot_log_msg_pb2 import RobotLog, LogLevel`